README.adoc

```diff
@@ -1,8 +1,14 @@
 == Spring Cloud Local Deployer
 
-WARNING: Spring Cloud Local Deployer is strictly only meant to be used for development. This 
-implementation is not recommended for production use because there’s no high availability, fault-tolerance, 
-or resiliency mechanics; hence, there are no reliable execution guarantees associated with it.
+Spring Cloud Local Deployer is an implementation of the Spring Cloud Deployer SPI for use
+to deploy applications on the same machine.  This occurs by this application spawning a
+new JVM process for the deployed application.
+
+NOTE:  It's important to note that this deployer spawns new JVMs that are not monitored
+or maintained by this deployer.  No attempts at high availability, fault tolerance, or
+resiliency are provided by the deployer.  Any use of this deployer in a production
+environment should be accompanied with additional monitoring at the app level (the apps
+this deployer deploys) to provide production grade resiliency.
 
```

Comment:
Maybe add a WE in the last sentence like: "This deployer will not be updated to take on those requirements and WE encourage users to explore the CloudFoundry and Kubernetes variants as ways to meet them."